nslsii/__init__.py

```diff
@@ -256,30 +259,37 @@ def configure_bluesky_logging(ipython):
     ----------
     ipython: InteractiveShell
         IPython InteractiveShell used to attach bluesky log handler to ipython
+    appdirs_appname: str
+        appname passed to appdirs.user_log_dir(); use the default for production,
+        set to something else for testing
 
     Returns
     -------
-    bluesky_log_file_path: str
+    bluesky_log_file_path: Path
         log file path
 
     """
     global bluesky_log_file_path
 
     if "BLUESKY_LOG_FILE" in os.environ:
+        bluesky_log_file_path = Path(os.environ["BLUESKY_LOG_FILE"])
         print(
             "bluesky log file configured from environment variable BLUESKY_LOG_FILE",
             file=sys.stderr,
         )
-        bluesky_log_file_path = os.environ["BLUESKY_LOG_FILE"]
     else:
+        bluesky_log_file_path = Path(
+            appdirs.user_log_dir(appname=appdirs_appname)
+        ) / Path("bluesky.log")
         print(
-            f"environment variable BLUESKY_LOG_FILE is not set, using default file path",
+            "environment variable BLUESKY_LOG_FILE is not set, using default file path",
```

Comment:
I'd print the default file path here, just to be explicit.

Reply:
I was printing it and then I noticed ipython also prints it, so I removed mine.

But I think you are right that this is also a good place to display the file path so I have added it.

Reply:
Thanks.